bears/stylus/StylintBear.py

```diff
@@ -36,12 +42,242 @@ class StylintBear:
+            ``&:hover`` won't count towards the limit.
+        :param check_duplicates:
+            Checks if selector or property duplicated unnecessarily.
+        :param efficient_properties:
+            Check for places where properties can be written more efficiently.
+            Supported values are ``True`` and ``False``.
+        :param extend_preference:
+            Pass in either ``@extend`` or ``@extends`` and then enforce that.
+            Both are valid in Stylus. It doesn't really matter which one
+            you use.
+            For example: If set to ``@extends``, prefer
+            ``@extends $some-var`` over ``@extend $some-var``
+            For example: If set to ``@extend:``, prefer
+            ``@extend $some-var`` over ``@extends $some-var``
+        :param indent_size:
+            This works in conjunction with ``max_selector_depth``. If you
+            indent with spaces this is the number of spaces you indent with.
+            If you use hard tabs, set this value to ``0``.
+            For example: If set to ``2``, prefer
+            ``/s/smargin: 0`` over ``/s/s/smargin: 0``
+        :param leading_zero:
+            When ``True``, prefer leading zeroes on decimal points. When
+            ``False``, disallow leading zeroes on decimal points.
+            Supported values are ``True`` and ``False``.
+        :param max_errors:
+            Set maximum number of Errors. If ``0``, all errors will
+            be shown without a limit.
+        :param max_warnings:
+            Set maximum number of Warnings. If ``0``, all errors will
+            be shown without a limit.
+        :param mixed_spaces_and_tabs:
+            If a non-negative number is passed to ``indent_size``, it assumes
+            soft tabs (i.e., spaces), and if ``0`` is passed to ``indent_size``
+            it assumes hard tabs.
+            For example: If ``indent_size = 4`` and
+            ``mixed_spaces_and_tabs = True``, prefer
+            ``s\s\s\smargin\s0`` over ``\tmargin\s0``
+            For example: If ``indent_size = 0`` and
+            ``mixed_spaces_and_tabs = True``, prefer
+            ``\tmargin\s0`` over ``\s\s\s\smargin\s0``
+        :param variable_naming_convention:
+            Enforce a particular naming convention when declaring variables.
+            Throws a warning if you don't follow the convention.
+            Supported values are ``lowercase-dash``, ``lowercase_underscore``,
+            ``camelCase``, ``BEM`` and ``False``.
+        :param strict_naming_convention:
+            By default, ``variable_naming_convention`` only looks at variable
+            names. If ``strict_naming_convention`` is set to ``True``,
+            ``variable_naming_convention`` will  also look at class and
+            id names.
+        :param none_keyword:
+            If ``True`` check for places where ``none`` used instead of 0. If
+            ``False`` check for places where 0 could be used instead of none.
```

Comment:
none not inside double backticks :)

Maybe you can also place 0 into double backticks, as it refers to a value in stylint code :+1: